bambam.py

```diff
@@ -217,14 +217,24 @@ def print_letter(self, char):
         self.screen.blit(text, textpos)
 
     def glob_data(self, pattern):
-        return glob.glob(os.path.join(self.progInstallBase, 'data', pattern))
+        fileList = []
+        for dataDir in self.dataDirs:
+            fileList.extend(glob.glob(os.path.join(dataDir, pattern)))
+        return fileList
 
     def run(self):
         """
         Main application entry point.
         """
         self.progInstallBase = os.path.dirname(os.path.realpath(sys.argv[0]))
 
+        self.dataDirs = [os.path.join(self.progInstallBase, 'data')]
+        xdg_data_home = os.getenv('XDG_DATA_HOME', os.path.expanduser("~/.local/share"))
```

Comment:
Please use single quotes here and on the next line. I know we're not 100% consistent yet, but we're getting there...

Reply:
ok